### ISSUE TITLE
'Files' document action shown for file types that aren't appropriate

### ISSUE THREAD

**Issue by jesusbagpuss** [eprints/eprints#336]
The 'files' actions are more relevant to e.g. html pages (where you need to include nested CSS/JS).
Currently it is shown for all documents - and e.g. for a PDF you *can* upload an additional file that makes no sense.

Rework EPrints::Plugin::Screen::EPrint::Document::Files to just render when file is not a complex type.

**Comment by sebastfr** [comment]
you guys should get rid of documents altogether - what's the point of supporting html docs on research repositories... 

ps: hope you're well john! :)

**Comment by jesusbagpuss** [comment]
We've got an awesome ethesis that was written as a wordpress site, exported as a flat html site and works great in EPrints :o)

I'm good - I hear you'll have a new colleague sometime soon!

**Comment by sebastfr** [comment]
yeah and then balance out that anecdote with all the times documents/files have been an issue for devs & users ;) 

oh i see the news reached leeds :)
